contrib/pkg/broker/server/server_test.go

```diff
@@ -117,12 +119,12 @@ func readJSON(rr *httptest.ResponseRecorder) (map[string]interface{}, error) {
 type Controller struct {
 	t *testing.T
 
-	catalog               func() (*brokerapi.Catalog, error)
-	getServiceInstance    func(id string) (string, error)
-	createServiceInstance func(id string, req *brokerapi.CreateServiceInstanceRequest) (*brokerapi.CreateServiceInstanceResponse, error)
-	removeServiceInstance func(id string) (*brokerapi.DeleteServiceInstanceResponse, error)
-	bind                  func(instanceID string, bindingID string, req *brokerapi.BindingRequest) (*brokerapi.CreateServiceBindingResponse, error)
-	unBind                func(instanceID string, bindingID string) error
+	catalog                         func() (*brokerapi.Catalog, error)
+	getServiceInstanceLastOperation func(id string) (*brokerapi.LastOperationResponse, error)
```

Comment:
I don't see a reason to leave out async support -- since it's OSB spec compliant -- even if this (test) broker doesn't support async (de)provision right now. It's possible, for example, that we could add it in the future, for purposes of testing.